src/App.tsx

```diff
@@ -1,27 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const handlerAll = () => {
+    getAll()
+      .then(todosFromServer => {
+        setGoods(todosFromServer);
+      })
+      .catch(() => {
+        setGoods([]);
```

Comment:
in catch block your should set the error into state and show it somewhere in your UI, so user will be aware of what happened wrong